files/anope/services.conf.j2

```diff
@@ -10,7 +10,7 @@ uplink
 	ipv6 = no
 	ssl = yes
 	port = 6697
-	password = "8o3hr13ohr9138eej2319r82h13e8j0"
+	password = "{{ vault_irc.link_keys.services }}"
```

Comment:
lolwut

Reply:
Yeah, that's kinda embarassing.
I noticed because I ran `irc.yml` and Ansible happily told me there were no changes on `services.irc.#!`...